src/Sulu/Component/SmartContent/Tests/Unit/Orm/DataProviderRepositoryTraitTest.php

```diff
@@ -103,12 +103,12 @@ public function __construct($accessControlQueryEnhancer, $queryBuilder)
                 $this->queryBuilder = $queryBuilder;
             }
 
-            public function createQueryBuilder()
+            public function createQueryBuilder($alias, $indexBy = null)
```

Comment:
Need to be compatbile to: https://github.com/sulu/sulu/blob/58d524cf1d6dac74713a916be30e273a34bc955c/src/Sulu/Component/SmartContent/Orm/DataProviderRepositoryTrait.php#L327-L343